light-client/src/lib.rs

```diff
@@ -121,6 +122,47 @@ impl LightClient for RemoteLightClient {
     }
 }
 
+#[derive(Clone)]
+pub struct UdpLightClient {
```

Comment:
I'd suggest moving this to its own file

Reply:
Sure, I've just left it here because the HTTP client was here as well. We can move them both to separate files